examples/client.rs

```diff
@@ -29,27 +29,22 @@ async fn execute_request(client: client::Client) -> io::Result<String> {
         .map_err(|err| io::Error::new(io::ErrorKind::Other, err))
 }
 
-// Compatibility with older tokio v0.2.x used by actix web v3. Not necessary with actic web v4.
-fn tokio_exporter_compat<T: SpanExporter + 'static>(exporter: T) -> BatchSpanProcessor {
-    let spawn = |fut| tokio::task::spawn_blocking(|| futures::executor::block_on(fut));
-    BatchSpanProcessor::builder(
-        exporter,
-        spawn,
-        tokio::time::delay_for,
-        tokio::time::interval,
-    )
-    .build()
-}
-
 #[actix_web::main]
 async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
     global::set_text_map_propagator(TraceContextPropagator::new());
     let exporter = opentelemetry_jaeger::new_pipeline()
```

Comment:
this can now be simplified back to:

```rust
let (_tracer, _uninstall) = opentelemetry_jaeger::new_pipeline()
    .with_service_name("actix_client")
    .install()
    .expect("pipeline install error");
```

don't need to do the manual builder and global assignment